test/integration/cmds/config/add.test.js

```diff
@@ -32,6 +34,16 @@ test('config add throws error when option mt is empty', done => {
     .end(done)
 })
 
+test('config add allows insecure', (done) => {
+  app()
+    .run('config add --insecure=true')
+    .code(0)
+    .end(() => {
+      const context = readSync('$HOME/.contentfulrc.json')
```

Comment:
do you mean readfilesync maybe? this is failing because it expects first arg to be an integer

Reply:
bah good catch